changelog.md

```diff
@@ -1,110 +1,139 @@
+# 0.4
+
+* ADD #204: SMAC now always saves runhistory files as `runhistory.py`.
+* MAINT #205: the SMAC repository now uses codecov.io instead of coveralls.io.
+* ADD #83: support of ACLIB 2.0 parameter configuration space file.
+* FIX #206: instances are now explicitly cast to `str`, instead the instances
```

Comment:
What does "instead the instances are `None`. You mean: "if not instances are given, a single instance called 'None' is used"